src/platform/forms/constants.js

```diff
@@ -24,6 +24,7 @@ export const VA_FORM_IDS = Object.freeze({
   FORM_21_4142: '21-4142',
   FORM_21_526EZ: '21-526EZ',
   FORM_21_686C: '686C-674',
+  FORM_21_686CV2: '686C-674V2',
```

Comment:
does it matter that this is `686C-674V2` as opposed to `686C-674-V2` ?

Reply:
I pushed a change for this. Tested everything and it's working as expected